src/pages/ExplorePage.tsx

```diff
@@ -6,34 +6,50 @@ import { AuthenticatedPageProps, Movie, User } from "../types";
 import ExploreMovieList from "../components/ExploreMovieList";
 import { fetchExploreMovies } from "../network/requests";
 
-import { Container, Header } from "semantic-ui-react";
+import { Container, Dropdown, Header } from "semantic-ui-react";
 
 const ExplorePage = ({
   sessionUser,
   setSessionUser
 }: AuthenticatedPageProps): JSX.Element => {
   const [movies, setMovies] = useState<Array<Movie>>([]);
-  const doFetchExploreMovies = () => fetchExploreMovies(setMovies);
+  const [sortMethod, setSortMethod] = useState("");
```

Comment:
Would be nice to do an enum here so it's clear what the options are. Also feels a bit weird to have the null case be an empty string, but 🤷‍♀ not a big deal